Musician Module/Controls/SheetButton.cs

```diff
@@ -0,0 +1,219 @@
+
+            if (this.MouseOver && relPos.Y > this.Height - BOTTOMSECTION_HEIGHT) {
+
+                this.MouseOverPreview = relPos.X < (SHEETBUTTON_WIDTH - 36 + 32) && relPos.X > (SHEETBUTTON_WIDTH - 36);
+                this.MouseOverPlay = relPos.X < (SHEETBUTTON_WIDTH - 73 + 32) && relPos.X > (SHEETBUTTON_WIDTH - 73);
+                this.MouseOverEmulate = relPos.X < (SHEETBUTTON_WIDTH - 109 + 32) && relPos.X > (SHEETBUTTON_WIDTH - 109);
+            } else {
+
+                this.MouseOverPlay = false;
+                this.MouseOverEmulate = false;
+                this.MouseOverPreview = false;
+            }
+
+            if (this.MouseOverPlay)
+            {
+
+                this.BasicTooltipText = $"Practice mode (Synthesia)";
+
+            }
+            else if (this.MouseOverEmulate)
+            {
+
+                this.BasicTooltipText = $"Emulate keys (Autoplay)";
+
+            }
+            else if (this.MouseOverPreview)
+            {
+                this.BasicTooltipText = "Preview";
+            } else {
+                this.BasicTooltipText = this.Title;
+            }
+        }
+
+        protected override CaptureType CapturesInput() {
+            return CaptureType.Mouse | CaptureType.Filter;
+        }
+
+        public override void PaintBeforeChildren(SpriteBatch spriteBatch, Rectangle bounds)
+        {
+            int iconSize = this.IconSize == DetailsIconSize.Large ? SHEETBUTTON_HEIGHT : SHEETBUTTON_HEIGHT - BOTTOMSECTION_HEIGHT;
+
+            // Draw background
+            spriteBatch.DrawOnCtrl(this, BackgroundSprite, bounds, Color.Black * 0.25f);
+
+            // Draw bottom section (overlap to make background darker here)
+            spriteBatch.DrawOnCtrl(this, BackgroundSprite, new Rectangle(0, bounds.Height - BOTTOMSECTION_HEIGHT, bounds.Width - BOTTOMSECTION_HEIGHT, BOTTOMSECTION_HEIGHT), Color.Black * 0.1f);
+
+            // Draw preview icon
+            if (this._mouseOverPreview)
+            {
+                if (this.IsPreviewing) {
+                    spriteBatch.DrawOnCtrl(this, Content.GetTexture("glow_stop"), new Rectangle(SHEETBUTTON_WIDTH - 36, bounds.Height - BOTTOMSECTION_HEIGHT + 1, 32, 32), Color.White);
+                } else {
+                spriteBatch.DrawOnCtrl(this, Content.GetTexture("glow_play"), new Rectangle(SHEETBUTTON_WIDTH - 36, bounds.Height - BOTTOMSECTION_HEIGHT + 1, 32, 32), Color.White);
+                }
+            }
+            else
+            {
+                if (this.IsPreviewing)
+                {
+                    spriteBatch.DrawOnCtrl(this, Content.GetTexture("stop"), new Rectangle(SHEETBUTTON_WIDTH - 36, bounds.Height - BOTTOMSECTION_HEIGHT + 1, 32, 32), Color.White);
+                }
+                else
+                {
+                    spriteBatch.DrawOnCtrl(this, Content.GetTexture("play"), new Rectangle(SHEETBUTTON_WIDTH - 36, bounds.Height - BOTTOMSECTION_HEIGHT + 1, 32, 32), Color.White);
```

Comment:
Looks like a lot of textures are pulled dynamically during the draw phase.  This isn't _too_ bad with the original `ContentService` since everything was cached, but even then, it's not as fast as it probably should be for the draw phase.  You'll want to cache these once you're using the `ContentsManager` as it does not have built in caching (since most textures are cached by their reference while in use, anyways).  You can see how most (I haven't finished converting all of them) of the built-in controls now maintain their reference to whatever textures they need: https://github.com/blish-hud/Blish-HUD/blob/22758abba30dbf13478a4405c1954265e04c2377/Blish%20HUD/Controls/TrackBar.cs#L11-L21